docs/site/Authentication-component-decorator.md

```diff
@@ -73,7 +83,7 @@ export class WhoAmIController {
 An example of the decorator when options **are** specified looks like this:
 
 ```ts
-@authenticate('basic', { /* some options for the strategy */})
+@authenticate({'basic', { /* some options for the strategy */}})
```

Comment:
`{name: 'basic, options: { /* some options for the strategy */ }}`

Reply:
good catch 👍